### ISSUE TITLE
/usr/local/lib/python3.7/dist-packages/simpletransformers/seq2seq/seq2seq_model.py:773: FutureWarning: Non-finite norm encountered in torch.nn.utils.clip_grad_norm_; continuing anyway. Note that the default behavior will change in a future release to error out if a non-finite total norm is encountered. At that point, setting error_if_nonfinite=false will be required to retain the old behavior.   model.parameters(), args.max_grad_norm /usr/local/lib/python3.7/dist-packages/torch/optim/lr_scheduler.py:134: UserWarning: Detected call of `lr_scheduler.step()` before `optimizer.step()`. In PyTorch 1.1.0 and later, you should call them in the opposite order: `optimizer.step()` before `lr_scheduler.step()`.  Failure to do this will result in PyTorch skipping the first value of the learning rate schedule. See more details at https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate   "https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate", UserWarning)

### ISSUE THREAD

**Issue by sarangtamrakar** [ThilinaRajapakse/simpletransformers#1199]
**Describe the bug**
You have used lr_schedular before the optimizer....

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**
 - OS

**Additional context**

**Comment by mfilan** [comment]
Hi,did you manage to resolve this problem?
